packages/api/src/providers/ActivityAcknowledgement/ActivityAcknowledgementComposer.tsx

```diff
@@ -9,9 +7,9 @@ import useActivityKeys from '../ActivityKeyer/useActivityKeys';
 import usePrevious from '../../hooks/internal/usePrevious';
 import useValueRef from '../../hooks/internal/useValueRef';
 
-import type { ActivityAcknowledgement } from './private/types';
+import { type ActivityAcknowledgement } from './private/types';
 
-type ActivityAcknowledgementComposerProps = PropsWithChildren<{}>;
+type ActivityAcknowledgementComposerProps = Readonly<PropsWithChildren<{}>>;
```

Comment:
This is my bad and lack of understanding about TypeScript couple of years ago. Could you help change this line to:

```suggestion
type ActivityAcknowledgementComposerProps = Readonly<{ children?: ReactNode | undefined }>;
```